server/src/main/java/io/crate/metadata/IndexReference.java

```diff
@@ -99,7 +105,7 @@ public IndexReference(int position,
                           IndexType indexType,
                           List<Reference> columns,
                           @Nullable String analyzer) {
-        super(ident, RowGranularity.DOC, DataTypes.STRING, ColumnPolicy.DYNAMIC, indexType, false, true, position, null);
+        super(ident, RowGranularity.DOC, DataTypes.STRING, ColumnPolicy.DYNAMIC, indexType, false, false, position, null);
```

Comment:
test was failing because of this artificial true value. whereas computed on mapping is false
```
IndexMetadata indexMetadata = clusterService.state().metadata().indices().valuesIt().next();
 Map<String, Object> sourceAsMap = indexMetadata.mapping().sourceAsMap();
assertThat(Maps.getByPath(sourceAsMap, "properties.title_desc_fulltext")).isEqualTo(mapping);
```

FT(IndexReference) is not a regular column, I think the way we compute it on mapping creation is correct
```
 if (definition.docValues() != definition.dataType.storageSupport().getComputedDocValuesDefault(definition.indexType)) {
            // definition.docValues falls back to default if not specified.
            // If computed value is non-default it means doc values are supported but disabled.
            mapping.put(DOC_VALUES, "false");
        }
```
        
and thus `IndexReference` ctor should follow that.